Jenkinsfile

```diff
@@ -12,6 +12,28 @@ CQZ_BUILD_ID = new Date().format('yyyyMMddHHmmss')
 def jobs = [:]
 def helpers
 
+def withRVM(version, cl) {
+    RVM_HOME='$HOME/.rvm'
+    paths = [
+        "$RVM_HOME/bin",
+        "${env.PATH}"
+    ]
+    
+    def path = paths.join(':')
```

Comment:
in this case I'm not sure if this syntax is more readable, how about: `def path = "${RVM_HOME}/bind:${env.PATH}"`